h/views/activity.py

```diff
@@ -21,6 +20,21 @@
 PAGE_SIZE = 200
 
 
+def pretty_link(url):
```

Comment:
This was in h.links but only used here, :shrug: I could a function in the service like the others but it's a bit different in the intent.